js/turtledefs.js

```diff
@@ -14,7 +14,7 @@ const LOGOJA1 = '<?xml version="1.0" encoding="UTF-8" standalone="no"?> <svg xml
="display:inline;fill:none;fill-opacity:1;fill-rule:evenodd;stroke:#505050;stroke-width:0.1046099;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" /> <path d="m 17.114016,8.5098241 a 0.94989708,0.58640587 78.078062 0 1 -0.340613,1.0406955 0.94989708,0.58640587 78.078062 0 1 -0.776562,-0.678756 0.94989708,0.58640587 78.078062 0 1 0.23956,-1.1290216 0.94989708,0.58640587 78.078062 0 1 0.807736,0.5318372 l -0.503878,0.3563839 z" id="path4265" style="display:inline;fill:#505050;fill-opacity:1;stroke-width:0.10460994;stroke-miterlimit:4;stroke-dasharray:none" /> <path d="M 20.413977,8.0315906 A 0.85676325,0.52891095 78.078062 0 1 20.10676,8.9702498 0.85676325,0.52891095 78.078062 0 1 19.406336,8.3580431 0.85676325,0.52891095 78.078062 0 1 19.622407,7.3397176 0.85676325,0.52891095 78.078062 0 1 20.350948,7.8194108 l -0.454474,0.3214416 z" id="path4265-2" style="display:inline;fill:#505050;fill-opacity:1;stroke-width:0.10460994;stroke-miterlimit:4;stroke-dasharray:none" /> <path id="path5720" d="m 21.134832,7.6963634 c -0.112318,-0.027757 -0.262497,-0.081054 -0.333731,-0.1184383 -0.144005,-0.075573 -0.299329,-0.2698653 -0.299329,-0.374426 0,-0.096607 -0.193298,-0.846814 -0.294133,-1.1415597 C 19.91785,5.2148827 19.426736,4.6758205 18.806808,4.5243423 18.574543,4.4675893 18.37796,4.3777172 18.37796,4.3282851 c 0,-0.1165874 0.518787,-0.372059 0.755587,-0.3720818 0.225129,-2.09e-5 0.551773,0.1955105 0.754007,0.4513556 0.08958,0.113326 0.336843,0.5587874 0.549476,0.9899141 0.630891,1.2791719 1.127464,1.9684738 1.567563,2.1759633 0.217308,0.1024518 0.226116,0.111942 0.130881,0.1410215 -0.159835,0.048804 -0.77495,0.037681 -1.000642,-0.018094 z" style="fill:#000000;fill-opacity:0;stroke-width:0.05230495;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none" /> <path id="path4245" d="m 15.544387,4.3143709 c 0,0 1.555226,2.1088053 2.078276,2.2761811 0.523049,0.1673759 0.550099,-1.2673939 0.550099,-1.2673939 0,0 0.01046,-0.8054962 -0.03138,-1.16117" style="fill:none;fill-rule:evenodd;stroke:none;stroke-width:0.1046099;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" /> <path id="path4249" d="m 18.944377,4.5456262 c 0.250182,0.02965 0.853235,-0.055903 1.134665,-0.7723694" style="fill:none;fill-rule:evenodd;stroke:#505050;stroke-width:0.1046099;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" /> <text id="text4245" y="2.0512714" x="11.557299" style="font-style:normal;font-weight:normal;font-size:0.12553188px;line-height:0%;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.01046099px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1" xml:space="preserve"><tspan style="font-size:0.4184396px;line-height:1.25;stroke-width:0.01046099px" y="2.0512714" x="11.557299" id="tspan4247"> </tspan></text> </g> </svg>';
```

Comment:
What is this about??? Did you intend to make a change here?

Reply:
Nope, might be the code editor (VSCode) I'm using who caused this change. Is there a style guide for `musicblocks`? If so I can tweak my settings to fit it.

Reply:
I don't think anyone ever wrote up coding standards for JS, which was always an afterthought for Sugar. Maybe it is time to do so.